scheduler/pkg/agent/v2.go

```diff
@@ -116,6 +116,7 @@ func getV2GrpcConnection(host string, plainTxtPort int) (*grpc.ClientConn, error
 
 	opts := []grpc.DialOption{
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
+		grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(util.GrpcMaxMsgSizeBytes), grpc.MaxCallSendMsgSize(util.GrpcMaxMsgSizeBytes)),
```

Comment:
do we need to set this for control plane operations?

Reply:
It's not a bad thing to be explicit about what our defaults are, I think, rather than relying on whatever a library happens to set (which may or may not make sense for us, or may not even be set).  With that said, I'd generally expect the control plane to use shorter messages, so wouldn't expect the same maxima.

I'm assuming this value might be used when creating buffers.  I'd imagine smaller buffers are used when possible and expanded up to the maximum, but just in case we might want to use a lower value to avoid guzzling too much memory.

Reply:
will remove here